pages/onboarding_cluster.ipynb

```diff
@@ -0,0 +1,248 @@
+    "### Recipe\n",
+    "If you want to know more about the overall design please consult the ADR documentation at [operate-first/support](https://github.com/operate-first/blueprint).\n",
+    "In general we store all the cluster-scoped resources in a `cluster-scope` kustomize application within this repository.\n",
+    "#### Adding namespaces\n",
+    "For easier onboard to ArgoCD later on, we prefer to follow a name pattern for all our namespaces. Please use your team name as a prefix to the namespace name like so: `$OWNER_TEAM-example-project`.\n",
+    "#####  Base resources\n",
+    "\n",
+    "Please run:\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!opfcli create-project $NAMESPACE_NAME $OWNER_TEAM -d $OPTIONAL_PROJECT_DESCRIPTION"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "This command will create:\n",
+    "\n",
+    "- A namespace in `cluster-scope/base/core/namespaces/$NAMESPACE_NAME`\n",
+    "- A blank user group for the `$OWNER_TEAM` if it does not exist yet in the `cluster-scope/base/user.openshift.io/groups/$OWNER_TEAM`\n",
+    "- An RBAC component for the project admin role `RoleBinding` in `cluster-scope/components/namespace-admin-rolebinding/$OWNER_TEAM` and maps it to the newly added namespace.\n",
+    "\n",
+    "##### Enabling namespace deployment to Zero cluster\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!cd cluster-scope/overlays/prod/$ENV/$TARGET_CLUSTER"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Open the `kustomization.yaml` file in this folder and add `../../base/core/namespaces/$NAMESPACE` to the `resources` field. We aim to keep this field alphabetically sorted for better human readability:\n",
+    "```\n",
+    "---\n",
+    "apiVersion: kustomize.config.k8s.io/v1beta1\n",
+    "kind: Kustomization\n",
+    "\n",
+    "resources:\n",
+    "...\n",
+    "- ../../base/core/namespaces/$NAMESPACE\n",
+    "...\n",
+    "```\n",
+    "##### Authenticate via OpenShift\n",
+    "Users are automatically created upon login to the appropriate cluster. This is done by logging in via SSO (any gmail account will work).\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "For the `MOC/Zero` you can access the login from this [link](https://console-openshift-console.apps.zero.massopen.cloud/).\n",
+    "This will lead the user to an authentication provider selection screen:\n",
+    "\n",
+    "```{image} ../img/openshift-login.png\n",
+    ":alt: openshift-login\n",
+    ":align: center\n",
+    "```\n",
+    "<br/><br/>\n",
+    "Please select `operate-first` provider.\\\n",
+    "With the user now created, we will need to provide them with appropriate rbac access to this namespace.\n",
+    "\n",
+    "#### Giving Users Rbac Permissions\n",
```

Comment:
Rename to "Add Users" or "Populate your team" or something.. RBAC may mean nothing to them.

Reply:
Resolved in latest commit.